Overview.src.html

```diff
@@ -592,11 +592,21 @@ <h4 id=terminology-headers>Headers</h4>
 <dfn title=concept-request-destination>destination</dfn>, which is one of
 the empty string,
 "<code title>document</code>",
+"<code title>embed</code>",
+"<code title>fetch</code>",
+"<code title>font</code>",
+"<code title>image</code>",
+"<code title>object</code>",
+"<code title>manifest</code>",
+"<code title>media</code>",
 "<code title>serviceworker</code>",
+"<code title>script</code>",
+"<code title>style</code>",
 "<code title>sharedworker</code>",
```

Comment:
"sharedworker" would should appear before "style", if we want to keep this alphabetical.